src/constants.F90

```diff
@@ -379,7 +379,8 @@ module constants
        FILTER_SPH_HARMONICS  = 18, &
        FILTER_SPTL_LEGENDRE  = 19, &
        FILTER_ZERNIKE        = 20, &
-       FILTER_PARTICLE       = 21
+       FILTER_PARTICLE       = 21, &
+       FILTER_ZERNIKE_RADIAL = 22
```

Comment:
Let's switch the numbers and locations for `FILTER_PARTICLE` and `FILTER_ZERNIKE_RADIAL`.  It's better to keep them close to each other since they're so similar, and the particle filter is so new that it won't mess anything up to make the change.